distribution/build.py

```diff
@@ -120,7 +120,13 @@ def get_dependencies():
         all_deps.extend(sorted(no_deps))  # No-deps installs
         all_deps.extend(sorted(no_cache))  # No-cache installs
 
-        return "\n".join(all_deps)
+        # Fix trustyai_fms package to use test PyPI index
```

Comment:
Nit: A `TODO` to remove the fix once the package is available on the main index would be nice

Reply:
+1 👍🏽